Source/Schema.NET/OneOrMany{T}.cs

```diff
@@ -15,131 +16,130 @@ namespace Schema.NET
         : IReadOnlyCollection<T>, IEnumerable<T>, IValues, IEquatable<OneOrMany<T>>
 #pragma warning restore CA1710 // Identifiers should have correct suffix
     {
-        private readonly List<T> collection;
-        private readonly T item;
+        private readonly T[] collection;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="OneOrMany{T}"/> struct.
         /// </summary>
         /// <param name="item">The single item value.</param>
         public OneOrMany(T item)
         {
-            this.collection = null;
-            this.item = item;
-        }
-
-        /// <summary>
-        /// Initializes a new instance of the <see cref="OneOrMany{T}"/> struct.
-        /// </summary>
-        /// <param name="array">The array of values.</param>
-        public OneOrMany(params T[] array)
-            : this(array is null ? null : new List<T>(array))
-        {
-        }
-
-        /// <summary>
-        /// Initializes a new instance of the <see cref="OneOrMany{T}"/> struct.
-        /// </summary>
-        /// <param name="collection">The collection of values.</param>
-        public OneOrMany(IEnumerable<T> collection)
-            : this(collection is null ? null : new List<T>(collection))
-        {
-        }
-
-        /// <summary>
-        /// Initializes a new instance of the <see cref="OneOrMany{T}"/> struct.
-        /// </summary>
-        /// <param name="list">The list of values.</param>
-        public OneOrMany(List<T> list)

```

Comment:
Removing this is a breaking change. I'm ok with that but this will be a major version upgrade.

Reply:
Yeah - its binary breaking though `List<T>` would still work to the `IEnumerable<T>` constructor if an application is recompiled.